api/src/main/java/org/apache/iceberg/PartitionField.java

```diff
@@ -68,11 +68,9 @@ public boolean equals(Object other) {
     }
 
     PartitionField that = (PartitionField) other;
-    return (
-        sourceId == that.sourceId &&
+    return sourceId == that.sourceId &&
```

Comment:
I'm okay with this, but I don't see a lot of benefit to removing unnecessary parens. If extra parens make something more readable (like this) or clarify order of operations even when matching the default, I would say we should keep them.

Reply:
I don't think the parens added readability - quite the opposite in fact, given that whenever I see parentheses I assume there would be some other clause in the boolean clause, e.g. `(...) && (...)`. It's a built-in rule in Checkstyle: http://checkstyle.sourceforge.net/config_coding.html#UnnecessaryParentheses. Doesn't look like it's strictly annotated in the style guide though: